bundles/io/org.eclipse.smarthome.io.net/META-INF/MANIFEST.MF

```diff
@@ -6,6 +6,7 @@ Bundle-Version: 0.9.0.qualifier
 Bundle-Vendor: Eclipse.org/SmartHome
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Import-Package: org.apache.commons.exec,
+ org.apache.commons.io,
```

Comment:
you should be able to remove this dependency again, if you follow my suggestion below.